uisdk/src/main/res/layout/uisdk_booking_checkout_view.xml

```diff
@@ -107,13 +110,44 @@
 
         </LinearLayout>
 
-    </androidx.core.widget.NestedScrollView>
+    </ScrollView>
+
+    <LinearLayout
+        android:id="@+id/bottomSectionContainer"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_gravity="bottom"
+        android:background="@drawable/kh_uisdk_checkout_bottom_layout">
+
+        <com.karhoo.uisdk.screen.booking.checkout.component.views.BottomPriceView
+            android:id="@+id/bottomPriceView"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_margin="@dimen/kh_uisdk_spacing_small"
+            android:layout_weight="0.3"/>
+
+        <View
+            android:layout_width="0dp"
+            android:layout_weight="0.2"
+            android:layout_height="0dp"/>
+
+        <com.karhoo.uisdk.base.view.LoadingButtonView
+            android:id="@+id/checkoutActionButton"
+            android:layout_width="0dp"
+            android:layout_weight="0.4"
+            android:layout_height="55dp"
+            android:layout_centerVertical="true"
+            android:layout_marginStart="@dimen/kh_uisdk_spacing_small"
```

Comment:
Since all the sides have the same margin, why not use `android:layout_margin` directly?

Reply:
Fixed